app/Formatters/Openinghours/JsonFormatter.php

```diff
@@ -22,6 +22,9 @@ class JsonFormatter extends BaseFormatter
     public function render($data)
     {
         $this->output = array_values($data);
+        if (count($this->output) === 1) {
```

Comment:
If a request to /api/v1/services returns only one service, we still want an array, so it's not sufficient to check how many items are in the output. We need to check if the request expects to get only one item, or if it expects to get 0, 1 or more items...